x-pack/plugins/security_solution/public/assistant/get_comments/stream/stream_observable.ts

```diff
@@ -72,9 +69,6 @@ export const getStreamObservable = (
       reader.cancel();
     };
   }).pipe(
-    // make sure the request is only triggered once,
-    // even with multiple subscribers
-    shareReplay(1),
```

Comment:
`shareReplay` causes the observable to be hot, which means unsubscribe does not actually stop the stream